Kitodo/src/main/resources/messages/messages_de.properties

```diff
@@ -23,6 +23,7 @@ Bibliothekarische\ Aufnahme\ f\u00FCr\ den\ lokalen\ OPAC=Bibliothekarische Aufn
 CatalogueChangeDocType=Dokumententyp ge\u00E4ndert zu
 CatalogueUnKnownType=Unbekannte Gattung
 CatalogueUsage=Verwendeter Katalog
+createMapping=ES Mapping erzeugen
```

Comment:
Please use full name for Elastic Search instead of the short name only understable by programmers.

Reply:
But this thing must be changed in future task when "real" button will be made.

Reply:
Is there a problem if we now use a good and readable name and it is switch to an other kind of visibleness in future?